src/tests/JIT/Regression/JitBlue/Runtime_93698/Runtime_93698.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+using Xunit;
+
+public static class Runtime_93698
+{
+    [Fact]
+    public static void TestShiftLeftLogicalOvershift()
+    {
+        if (Sse.IsSupported)
```

Comment:
was it supposed to be Sse2.IsSupported?

Reply:
Yep, my mistake. This would have failed in the jitstress isa jobs.